rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html

```diff
@@ -0,0 +1,115 @@
+    </div>
+    <div class="col-xs-11">
+      <div>
+        <div class="row">
+          <div class="col-xs-11">
+            <h1>{{ record | person_label(current_i18n.language) }}</h1>
+            {{ build_row('MEF ID', record.pid) }}
+          </div>
+        </div>
+        <div class="clearfix margin-bottom-20">
+          <ul class="nav nav-pills pull-right" role="tablist">
+            <li role="presentation" class="active">
+              <a href="#merged" data-toggle="tab"><i class="fa fa-list-ul"></i></a>
+            </li>
+            <li role="presentation">
+              <a href="#separated" data-toggle="tab"><i class="fa fa-bars"></i></a>
+            </li>
+          </ul>
+        </div>
+        <div class="tab-content">
+          <div role="tabpanel" class="tab-pane active" id="merged">
+            {% include('rero_ils/_person_merged.html') %}
+          </div>
+          <div role="tabpanel" class="tab-pane" id="separated">
+            {% include('rero_ils/_person_separated.html') %}
+          </div>
+        </div>
+
+        <!-- JSON -->
+        {%- if current_user | can_edit %}
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="btn-group  pull-right" role="group" aria-label="...">
+              <a href="#source-{{ record.id }}" data-toggle="collapse" type="button" class="btn btn-info">
+                              <i class="fa fa-search-plus  fa-fw"></i> {{_('JSON')}}
+                          </a>
+            </div>
+          </div>
+        </div>
+        <div id="source-{{ record.id }}" class="collapse">
+          <pre>{{ record.dumps() | tojson_pretty }}</pre>
+        </div>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+</div>
+{%- endblock %}
+
+
+{% macro build_row(name, value) %}
+{% if value %}
+<div class="row">
+  <div class="col-xs-2">
+    {{_(name)}}
+  </div>
+  <div class="col-xs-9">
+    {% if value is string %}
+    {{ value }}
+    {% else %}
+    <ul class="list-unstyled">
+      {% for element in value %}
+      <li>{{ element }}</li>
+      {% endfor %}
+    </ul>
+    {% endif %}
+  </div>
+</div>
+{% endif %}
+{% endmacro %}
+
+{% macro build_row_merged(name, value) %}
+{% if value %}
+<div class="row margin-bottom-20">
+  <div class="col-xs-2">
+    {{_(name)}}
+  </div>
+  <div class="col-xs-9">
+    <ul class="list-unstyled">
+      {% for element in value %}
+      <li>
+        {{ element }}
+        {% for source in value[element] %}
+        <span class="label label-default text-uppercase">{{ source }}</span>
+        {% endfor %}
+      </li>
+      {% endfor %}
+    </ul>
+  </div>
+</div>
+{% endif %}
+{% endmacro %}
+
+{% block javascript %}
+{{ super() }}
+{% assets "rero_ils_tab_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+<script type="text/javascript">
+$(document).ready(function(){
+    $('a[data-toggle="tab"]').on('click', function(e) {
+        localStorage.setItem('activeTab', $(this).attr('href'));
```

Comment:
A small comment is welcome. Is it for storing the opened tab?

Reply:
ok